mtags/src/main/scala-2/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -242,11 +242,17 @@ trait MtagsEnrichments extends ScalametaCommonEnrichments {
         text(pos.end - 1) == '`' &&
         pos.start != (pos.end - 1) // for one character names, e.g. `c`
       //                                                    start-^^-end
+      lazy val backtickedSetter =
+        text(pos.start) == '`' && text.length > pos.end + 4 && text
+          .slice(pos.end + 1, pos.end + 4)
+          .mkString == "_=`"
```

Comment:
```suggestion
           == "_=`"
```
slice returns string already I think?

Reply:
`Array[Char]`

Reply:
`==` on `Array`s won't compare by elements

Reply:
ach right, let's leave it as is then